cap-helm-charts-ci/vars.yaml.template

```diff
@@ -0,0 +1,14 @@
+---
+ci-uri: https://github.com/SUSE/cap.git
+ci-branch: master
+bucket: <CAP-RELEASE-BUCKET> # S3 bucket to pick the release from.
+region: <CAP-RELEASE-BUCKET-REGION> # Region of the bucket.
+access-key-id: <S3-ACCESS-KEY-ID> # Access key for the release bucket.
+secret-access-key: <S3-SECRET-ACCESS-KEY> # Secret Key for the release bucket.
+helm-charts-repo-uri: <HELM-CHART-REPO-URI> # URI for helm chart repo.
+helm-charts-repo-branch: <HELM-CHART-REPO-BRANCH> # Branch for helm chart repo.
+helm-charts-repo-private-key: <HELM-CHARTS-REPO-PRIVATE-KEY> # Github private key for helm chart repo.
+github-token: <GITHUB-TOKEN> # Your GitHub token.
```

Comment:
For secrets, you could assume the user has the blackbox in `BLACKBOX_DIR` and get the values in a script.
Otherwise this is error prone, and might leak secrets all over the place, depending on the user's carefulness.

Reply:
Another option is adding the secret itself ala `((secret))` into the concourse.suse.dev instance (see [here](https://github.com/SUSE/cloudfoundry/tree/master/concourse.suse.dev#manipulating-concourse-pipeline-secrets)).

This has the benefit that the pipeline yaml doesn't contain a plain secret (which could be seen by `fly get-pipeline`), nor fly hijacking allows you to see the secret. But on the pipeline build, the value will be correctly used.